phoenix-core/src/it/java/org/apache/phoenix/end2end/CsvBulkLoadToolIT.java

```diff
@@ -62,7 +62,7 @@ public class CsvBulkLoadToolIT extends BaseOwnClusterIT {
 
     private static Connection conn;
     private static String zkQuorum;
-    private static final String filePath = System.getProperty("java.io.tmpdir");
```

Comment:
This is a HDFS path. not a local path.
This should NOT use java.io.tmpdir.

Please check each changed test  to make sure that only local paths use java.io.tmpdir, and not HadoopFS paths.